hack/pseudo-version/pseudo_version_tool_freshness.sh.in

```diff
@@ -31,7 +31,7 @@ platforms=(
 )
 
 for platform in "${platforms[@]}"; do
-  computed_hash=$(sha256sum "${pseudo_version_tools[$platform]}" | cut -d' ' -f1)
```

Comment:
While sha256sum is not cross platform, `shasum -a 256` isn't either. See also https://github.com/edgelesssys/constellation/blob/7c345f45037f13315eb6ac34ad8f25b7ae7e475f/tools/workspace_status.sh#L39-L43